src/renderer/utils/gamepad/index.ts

```diff
@@ -3,8 +3,8 @@ import { ITwistMsg } from '@/renderer/utils/ros/rosMsgs.types'
 import { Vector3 } from '@/renderer/utils/math/types'
 import { store } from '@/renderer/store/store'
 
-const deadzone = (value: number): number => {
-  const deadzone = 0.05
+export const deadzone = (value: number): number => {
+  const deadzone = 0.1
```

Comment:
Damn, c'est quand meme gros comme deadzone ca, mais j'imagine que c'est testé avec le robot so ¯\_(ツ)_/¯

Reply:
Oh yeah good catch, I changed it because the third person cam was harder to control with a lower deadzone, but I scaled down the movement since then so the deadzone should probably be fine at 0.05, I'll test it once we start testing in 2 days